src/cms/templates/_raw.html

```diff
@@ -9,25 +8,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, shrink-to-fit=no, user-scalable=no" />
     <title>{% trans 'Integreat Editorial System' %}</title>
     <link rel="shortcut icon" type="image/png" href="{% static 'images/favicon.ico' %}">
-    {% compress css %}
-    {% block css %}{% endblock %}
-    <link href="{% static 'css/style.min.css' %}" rel="stylesheet" type="text/css">
-    <link href="{% static 'css/upload_form.css' %}" rel="stylesheet" type="text/css">
-    <link href="{% static 'tailwindcss/dist/tailwind.css' %}" rel="stylesheet" type="text/css">
-    {% endcompress %}
-    {% compress js %}
-    <script src="{% static 'umbrellajs/umbrella.js' %}"></script>
-    <script src="{% static 'feather-icons/dist/feather.js' %}"></script>
-    <script src="{% static 'base64-js/base64js.min.js' %}"></script>
+
+    <script src="{% static 'main.js' %}"></script>
+    <link href="{% static 'main.css' %}" rel="stylesheet" type="text/css">
+
     {% block javascript_head %}{% endblock %}
-    {% endcompress %}
 </head>
 <body>
+    <div data-do-something></div>
```

Comment:
When is this div used? I couldn't find any occurrences of `do-something` anywhere else in the code...

Reply:
You remember when we had a call and @ulliholtgrave asked how to add a new feature? It just does something 🦄

Reply:
I can't say we don't have a soft spot for obscure unicorn features :joy: 

https://github.com/Integreat/integreat-cms/blob/d576357cba25c0e40def05621d555e571f5e7ead/src/api/v3/regions.py#L130-L140